plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeSplitManager.java

```diff
@@ -214,4 +239,16 @@ private ConnectorSession testingConnectorSessionWithConfig(DeltaLakeConfig delta
                 .setPropertyMetadata(sessionProperties.getSessionProperties())
                 .build();
     }
+
+    @DataProvider
+    public static Object[][] absolutePaths()
+    {
+        return new Object[][] {{"file://path/to/a/table/directory/file", "file://path/to/a/table/directory/file"},
+                {"abfs://ct@st.dfs.core.windows.net/path/to/a/table/directory/file", "abfs://ct@st.dfs.core.windows.net/path/to/a/table/directory/file"},
+                {"hdfs://path/to/a/table/directory/file", "hdfs://path/to/a/table/directory/file"},
+                {"s3://my-s3-bucket/path/to/a/table/directory/file", "s3://my-s3-bucket/path/to/a/table/directory/file"},
+                {"gs://my-gcp-bucket/path/to/a/table/directory/file", "gs://my-gcp-bucket/path/to/a/table/directory/file"},
+                {"abfs://ct@st.dfs.core.windows.net/+ab+/a%25/a%2525/databricks-compatibility-test-table/directory/file",
```

Comment:
Let's shorten the path and improve the format. e.g.
```suggestion
        return new Object[][] {
                {"file://path/to/file", "file://path/to/file"},
                {"abfs://ct@st.dfs.core.windows.net/path/to/file", "abfs://ct@st.dfs.core.windows.net/path/to/file"},
                {"hdfs://path/to/file", "hdfs://path/to/file"},
                {"s3://my-s3-bucket/path/to/file", "s3://my-s3-bucket/path/to/file"},
                {"gs://my-gcp-bucket/path/to/file", "gs://my-gcp-bucket/path/to/file"},
                {"abfs://ct@st.dfs.core.windows.net/+ab+/a%25/a%2525/test/directory/file", "abfs://ct@st.dfs.core.windows.net/+ab+/a%/a%25/test/directory/file"}
        };
```

Reply:
done